sogs/model/message.py

```diff
@@ -42,6 +42,25 @@ def __init__(self, row=None, *, sender=None, recip=None, data=None):
         assert row is not None
         self._row = row
 
+    @staticmethod
+    def delete_all(*, recip=None, sender=None):
+        """Delete all messages sent to a user or from a user.
+        returns the number of rows affected.
+        """
+        assert recip or sender
+        recip_id = recip.id if recip else None
+        sender_id = sender.id if sender else None
+        result = query(
+            f"""
+            DELETE FROM inbox WHERE
+            {'recipient = :recip' if recip else '' }
+            {'sender = :sender' if sender else '' }
+            """,
```

Comment:
This query is going to be invalid if you pass in both a `recip` and ` sender` at the same time.